packages/code-editor/src/CodeEditor/CodeEditor.tsx

```diff
@@ -102,6 +104,7 @@ export const CodeEditor = forwardRef<CodeMirrorRef, CodeEditorProps>(
       const hyperLinkCompartment = new Compartment();
       const lineWrappingCompartment = new Compartment();
       const indentExtensionCompartment = new Compartment();
+      const tooltipCompartment = new Compartment();
```

Comment:
Super nit for later, but we may want to consider a data structure to contain all these compartments?